README.md

```diff
@@ -4,53 +4,36 @@ A clean, simple, and unobtrusive ruby authentication solution.
 
 [![Gem Version][5]][6] [![Build Status][1]][2] [![Code Climate][7]][8] [![Dependency Status][3]][4]
 
-## Sponsors
-
-[![Timber Logging](http://res.cloudinary.com/timber/image/upload/v1490556810/pricing/sponsorship.png)](https://timber.io?utm_source=github&utm_medium=authlogic)
-
-[Tail Authlogic users](https://timber.io/docs/app/console/tail-a-user) in your logs!
-
-## Documentation
-
-| Version     | Documentation |
-| ----------- | ------------- |
-| Unreleased  | https://github.com/binarylogic/authlogic/blob/master/README.md |
-| 4.3.0       | https://github.com/binarylogic/authlogic/blob/v4.3.0/README.md |
-| 3.7.0       | https://github.com/binarylogic/authlogic/blob/v3.7.0/README.md |
-| 2.1.11      | https://github.com/binarylogic/authlogic/blob/v2.1.11/README.rdoc |
-| 1.4.3       | https://github.com/binarylogic/authlogic/blob/v1.4.3/README.rdoc |
```

Comment:
The doc links have to stay at the top of the page. The goal is to prevent people from accidentally reading `master` docs, when they should be reading eg. 3.7 docs.